.github/workflows/presubmit.yml

```
@@ -112,6 +113,17 @@ jobs:
         if: ${{ matrix.android-abi != '' }}
         run: |
           echo "android-cmake-args=-DCMAKE_TOOLCHAIN_FILE=${{ github.workspace }}/ndk/android-ndk-r25c/build/cmake/android.toolchain.cmake -DANDROID_ABI=${{ matrix.android-abi }} -DVulkan_LIBRARY=${{ github.workspace }}/ndk/android-ndk-r25c/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/aarch64-linux-android/33/libvulkan.so" >> $GITHUB_ENV
+      - name: Build Perfetto
+        if: ${{ matrix.perfetto }}
+        shell: bash
+        env:
+          PERFETTO_DIR: '${{ github.workspace }}/external/perfetto'
```

```suggestion
          PERFETTO_DIR: ${{ github.workspace }}/external/perfetto
```

This might fix the issues you're seeing.